anitya/db/models.py

```diff
@@ -948,6 +948,31 @@ def get_id(self):
         """
         return six.text_type(self.id)
 
+    def __json__(self, detailed=False):
```

Comment:
It looks like ```detailed``` does not do anything (and it is also undocumented).

Reply:
Copied the method from other class and didn't noticed this parameter. Will remove.